app/experimenter/kinto/tests/test_tasks.py

```diff
@@ -73,6 +73,29 @@ def test_push_experiment_to_kinto_reraises_exception(self):
 
 
 class TestCheckKintoPushQueue(MockKintoClientMixin, TestCase):
+    def setUp(self):
+        super().setUp()
+        mock_dispatchee_task_patcher = mock.patch(
+            "experimenter.kinto.tasks.nimbus_check_kinto_push_queue_by_application.delay"
+        )
+        self.mock_dispatchee_task = mock_dispatchee_task_patcher.start()
+        self.addCleanup(mock_dispatchee_task_patcher.stop)
+
+    def test_dispatches_check_push_queue(self):
+        mock_application_collections = {
+            "FOO": "foo",
+            "BAR": "bar",
+        }
+
+        with mock.patch.object(
+            NimbusExperiment, "KINTO_APPLICATION_COLLECTION", mock_application_collections
+        ):
```

Comment:
Do you need to mock this out in the test?

Reply:
@jaredlockhart I think yes since the task doesn't care which applications it uses, only that it does something with them, and with this we can freely modify the applications while being confident the task is still doing its job.

Reply:
@jaredlockhart nvm I just realized how pointless is

Reply:
Classic case of me being excited that I learned something and looking for any opportunity to use it😜